.claude/skills/triaging-module-requests/analysis-templates.md

```diff
@@ -0,0 +1,352 @@
+# Analysis Comment Templates
+
+This document provides templates for analysis comments posted to module request issues. Customize based on priority score and specific circumstances.
+
+## General Template Structure
+
+```markdown
+## 📊 Module Request Analysis
+
+**Tool:** [Tool Name]
+**Repository:** [GitHub URL]
+**Priority Score:** [Score]/100 [Priority Band Emoji and Label]
+
+### Score Breakdown
+
+| Category              | Score | Max          | Notes |
+| --------------------- | ----- | ------------ | ----- |
+| Tool Popularity       | XX/25 | [Brief note] |
+| Package Downloads     | XX/15 | [Brief note] |
+| Community Engagement  | XX/35 | [Brief note] |
+| Request Quality       | XX/20 | [Brief note] |
+| Technical Feasibility | XX/15 | [Brief note] |
+
+### [Priority Band]: Recommendation
+
+[Specific feedback paragraph]
+
+### Next Steps
+
+[Specific actionable items]
+
+---
+
+_This analysis was performed automatically. For questions or to request re-analysis, comment `@claude analyze-module`._
+```
+
+## High Priority Template (≥70)
```

Comment:
We really only need one template here

Also I'd like it to be a one-liner, with everything else hidden within a `<description>` tag.